### ISSUE TITLE
Support React

### ISSUE THREAD

**Issue by Sphinxs** [gdsestimating/three-dxf#25]
Please, add support to React. If I install the `dxf-parse`, `three` and `three-dxf`, when I import all the modules in my React application:

```javascript
import DxfParser from 'dxf-parser';

import * as THREE from 'three';

import * as ThreeDxf from 'three-dxf';
```

I receive an error, `THREE is not defined`. This error happens because the module `three-dxf` expect the `THREE` on [global scope](https://github.com/gdsestimating/three-dxf/blob/23cdb7381b6a47a50db9fdc296f991bab0615cc0/dist/three-dxf.js#L11): 

```html
<script src='https://cdnjs.cloudflare.com/ajax/libs/three.js/101/three.js'></script>
```

To solve this issue I had import the Dxf Parse and Three Dxf on my React component and the Three in the `index.html` file. It could be cool to support React by default.

**Comment by bzuillsmith** [comment]
Good point. Been meaning to make it more versatile for a while but haven't done it yet. I just updated dxf-parser to be usable in any environment. I'll get to this library next.

**Comment by bzuillsmith** [comment]
I'm very close to supporting react I think. I added webpack to the project and updated the code to use es5 imports. The file that gets generated into the dist folder might not be react friendly. But I set up the library to point to the actual source in the case of something that supports es5 imports.

It'd be nice to package up everything into one es5 file in a way that react likes but there is a possibility it works as it is. I set up the package.json to point to the src/index.js file directly but I haven't really tested it. I'm just a react beginner so I'm not sure what more is required. Going to wait to publish until we figure this out.
